homestar-runtime/src/event_handler/event.rs

```diff
@@ -301,6 +303,34 @@ impl Event {
                     .dial(peer_id)
                     .map_err(anyhow::Error::new)?;
             }
+            Event::Bootstrap => {
+                if event_handler
```

Comment:
Could we add a comment here so we remember why we are checking for connected peers?